.env.rolling

```diff
@@ -6,8 +6,5 @@ VWO_ID=713156
 SKYLIGHT_ENV=dev
 FAST_IMAGE_TIMEOUT=0.25
 SIZE_IMAGES=1
-BIGQUERY_TABLE_NAME=events
```

Comment:
Shouldn't they be removed from the other .env files?

Reply:
Done.